app/models/manageiq/providers/ansible_tower/shared/automation_manager/tower_api.rb

```diff
@@ -8,7 +8,9 @@ def create_in_provider(manager_id, params)
       tower_object = provider_collection(manager).create!(params)
 
       refresh(manager)
-      find_by!(:manager_id => manager.id, :manager_ref => tower_object.id)
+      target = find_by!(:manager_id => manager.id, :manager_ref => tower_object.id)
+      refresh(target)
```

Comment:
There has to be a better way to do this than refreshing twice.

@agrare @Fryguy any ideas on this?  Can we leverage any of the work that the oVirt team did for event-based refreshes?